pkgs/applications/editors/vim/plugins/updater.nix

```diff
@@ -1,59 +1,8 @@
 {
-  lib,
-  buildPythonApplication,
-  makeWrapper,
-  nix,
-  nix-prefetch-git,
-  nurl,
+  writers,
   python3Packages,
-  vimPluginsUpdater,
-  writeShellScript,
-
-  # optional
-  neovim-unwrapped,
 }:
-buildPythonApplication {
-  pname = "vim-plugins-updater";
-  version = "0.1";
-
-  format = "other";
-
-  nativeBuildInputs = [
-    makeWrapper
-    python3Packages.wrapPython
-  ];
-
-  pythonPath = [
-    python3Packages.gitpython
-  ];
-
-  dontUnpack = true;
-
-  installPhase = ''
-    mkdir -p $out/bin $out/lib
-    cp ${./update.py} $out/bin/vim-plugins-updater
-    cp ${./get-plugins.nix} $out/bin/get-plugins.nix
-
-    # wrap python scripts
-    makeWrapperArgs+=( --prefix PATH : "${
-      lib.makeBinPath [
-        nix
-        nix-prefetch-git
-        neovim-unwrapped
-        nurl
-      ]
-    }" --prefix PYTHONPATH : "${./.}:${../../../../../maintainers/scripts/pluginupdate-py}" )
-    wrapPythonPrograms
-  '';
-
-  shellHook = ''
-    export PYTHONPATH=pkgs/applications/editors/vim/plugins:maintainers/scripts/pluginupdate-py:$PYTHONPATH
-  '';
-
-  passthru.updateScript = writeShellScript "updateScript" ''
-    # don't saturate the update bot connection
-    ${lib.getExe vimPluginsUpdater} --proc 2 update
-  '';
-
-  meta.mainProgram = "vim-plugins-updater";
-}
+writers.writePython3Bin "vim-plugins-updater" {
+  libraries = [ python3Packages.nixpkgs-updaters-library ];
+  doCheck = false;
```

Comment:
It'd be nice if there were some tests, but it seems the old package also didn't have a check phase

Reply:
> @khaneliman Does this block this PR? If so maybe an interim solution is needed and/or the follow up PR should be worked on before this is merged?

For me its a big one, https://github.com/NixOS/nixpkgs/commits/master/pkgs/applications/editors/vim/plugins/nvim-treesitter/generated.nix . Doesn't seem like they get updated any other time other than when I do these batch updates.